### ISSUE TITLE
Mog Locker Accessibility / Cannot Update Access to Vana'diel Lease

### ISSUE THREAD

**Issue by DeuceFFXI** [EdenServer/community#3502]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

Players are currently unable to update the accessibility of their mog locker with Fubruhn in Whitegate.  The option to switch from Al Zahbi only, to all three cities plus Jeuno is available; however, nothing occurs after selecting this option.  After saying "yes" to the Vana'diel Lease option, time remaining should be adjusted to 5/7s and accessibility should be extended to Jeuno, San d'Oria, Bastok, Windurst, and Nomad Moogles.

**Comment by DeuceFFXI** [comment]
Ignore the first post, but this is related.

It seems as though the NPC Fubruhn doesn't properly update when changing accessibility.  I'm uncertain if there is some dialogue missing or if the server just fails to calculate the proper time remaining and update the player.  Currently the setting will change, but it appears as though it were bugged, as there is no feedback given to the player.

**Comment by KnowOne134** [comment]
Closing this issue and making another about the time correction as original post is working
